package.json

```diff
@@ -2,6 +2,7 @@
   "name": "react-formal",
   "version": "0.17.9",
   "description": "Classy HTML form management for React",
+  "jsnext:main": "src/index.js",
```

Comment:
you are right we don't want bikeshedding anything, we don't need an indicator for general untranspiled code, the src folder signals enough. 

 JSNEXT is specifically for signally to environments and bundlers where the essential module versions of your code is as an interminable solution until transpilation of modules isn't  needed because they are natively supported.  using it however we feel defeats the usefulness and validity of the convention ppl are trying to establish to solve a specific problem.